src/pages/logExplorer/components/LogsViewer/Clustering/Table.tsx

```
@@ -0,0 +1,267 @@
+            return 0;
+          });
+          return <Space size={'small'}>{sortedParts.map((part) => {
+            if (part.type === 'pattern') {
+              if (backEndCluster) {
+                return (
+                  <PatternPopover key={part.part_id} uuid={(row.uuid)} partId={part.part_id} title={part.data}>
+                    <Tag className='mr-0 cursor-pointer' color='purple'>{part.data}</Tag>
+                  </PatternPopover>
+                );
+              } else {
+                return <Tag className='mr-0 cursor-pointer' color='purple'>{part.data}</Tag>
```

Biome (2.4.4)

[error] Missing key property for this element in iterable.

(lint/correctness/useJsxKeyInIterable)